modules/ui/UiOvertureInspector.js

```diff
@@ -0,0 +1,302 @@
+   * @param  {string} color - a hexstring like '#rgb', '#rgba', '#rrggbb', '#rrggbbaa'  (alpha values are ignored)
+   * @return {number} A number representing the perceived brightness
+   */
+  getBrightness(color) {
+    const short = (color.length < 6);
+    const r = parseInt(short ? color[1] + color[1] : color[1] + color[2], 16);
+    const g = parseInt(short ? color[2] + color[2] : color[3] + color[4], 16);
+    const b = parseInt(short ? color[3] + color[3] : color[5] + color[6], 16);
+    return ((r * 299) + (g * 587) + (b * 114)) / 1000;
+  }
+
+
+  /**
+   * renderFeatureInfo
+   * Renders the 'feature-info' section (the dataset name)
+   * @param {d3-selection} $selection - A d3-selection to a HTMLElement that this content should render itself into
+   */
+  renderFeatureInfo($selection) {
+    const datum = this.datum;
+    if (!datum) return;
+
+    const context = this.context;
+    const l10n = context.systems.l10n;
+    const rapid = context.systems.rapid;
+
+    const datasetID = datum.__datasetid__;
+    const dataset = rapid.datasets.get(datasetID);
+    const color = dataset.color;
+
+    let $featureInfo = $selection.selectAll('.feature-info')
+      .data([0]);
+
+    // enter
+    const $$featureInfo = $featureInfo.enter()
+      .append('div')
+      .attr('class', 'feature-info');
+
+    $$featureInfo
+      .append('div')
+      .attr('class', 'dataset-label');
+
+    // update
+    $featureInfo = $featureInfo.merge($$featureInfo);
+
+    $featureInfo
+      .style('background', color)
+      .style('color', this.getBrightness(color) > 140.5 ? '#333' : '#fff');
+
+    // Attempt to localize the dataset name, fallback to 'label' or 'id'
+    const text = dataset.labelStringID ? l10n.t(dataset.labelStringID) : (dataset.label || dataset.id);
+    $featureInfo.selectAll('.dataset-label')
+      .text(text);
+  }
+
+
+  /**
+   * renderPropertyInfo
+   * Renders the 'property-info' section
+   * @param {d3-selection} $selection - A d3-selection to a HTMLElement that this content should render itself into
+   */
+  renderPropertyInfo($selection) {
+    const properties = this.datum?.geojson.properties;
+    if (!properties) return;
+
+    const context = this.context;
+    const l10n = context.systems.l10n;
+
+    let $propInfo = $selection.selectAll('.property-info')
+      .data([0]);
+
+    // enter
+    const $$propInfo = $propInfo.enter()
+      .append('div')
+      .attr('class', 'property-info');
+
+    const $$propBag = $$propInfo
+      .append('div')
+      .attr('class', 'property-bag');
+
+
+      //Overture properties can come to us as strings, JSON arrays, or JSON objects. Handle all three!
+    for (const [k, v] of Object.entries(properties)) {
+      const $$propHeading = $$propBag
+      .append('div')
+      .attr('class', 'property-heading');
+
+      let key = k;
+
+      // Some params come to us via pmtiles with a prepended '@' sign.
```

Comment:
Is there a reason why they start with `@` ?  (I'm new to this and don't know.)

Reply:
Great question. That looks to be a specific issue to the way Overture maps is generating the tilesets, and is (for the moment) specific to the @name property. 

eg: https://github.com/search?q=repo%3AOvertureMaps%2Foverture-tiles%20%40name&type=code

This pattern is the same for the addresses pmtiles as well. let me do some digging with the folks over in those parts.